identity/src/main/java/com/android/identity/DataTransportWifiAware.java

```diff
@@ -519,8 +520,8 @@ void initiatorConnect(Network network, NetworkCapabilities networkCapabilities)
 
         // TODO: it's not clear whether port should be included here, we include it for now...
         //
-        mInitiatorIPv6HostString = String.format("[%s]:%d", strippedAddress.getHostAddress(),
-                peerPort);
+        mInitiatorIPv6HostString = String.format(Locale.US,
+                "[%s]:%d", strippedAddress.getHostAddress(), peerPort);
```

Comment:
I'm not sure we want to use locale when logging port number. IIUC, this is well understood to be an undecorated integer value (no commas or any formatting) around the world.